docs/core/tools/dotnet-new.md

```diff
@@ -58,714 +28,688 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
+  > [!NOTE]
+  > To uninstall a template using a `PATH`, you need to fully qualify the path. For example, *C:/Users/\<USER>/Documents/Templates/GarciaSoftware.ConsoleTemplate.CSharp* will work, but *./GarciaSoftware.ConsoleTemplate.CSharp* from the containing folder will not.
+  > Additionally, do not include a final terminating directory slash on your template path.
 
-`-i|--install <PATH|NUGET_ID>`
+- **`--update-apply`**
 
-Installs a source or template pack from the `PATH` or `NUGET_ID` provided. If you want to install a prerelease version of a template package, you need to specify the version in the format of `<package-name>::<package-version>`. By default, `dotnet new` passes \* for the version, which represents the last stable package version. See an example at the [Examples](#examples) section.
+  Checks if there are updates available for the template packs that are currently installed and installs them. Available since .NET Core 3.0 SDK.
 
-For information on creating custom templates, see [Custom templates for dotnet new](custom-templates.md).
+- **`--update-check`**
 
-`-l|--list`
+  Checks if there are updates available for the template packs that are currently installed. Available since .NET Core 3.0 SDK.
 
-Lists templates containing the specified name. If invoked for the `dotnet new` command, it lists the possible templates available for the given directory. For example if the directory already contains a project, it doesn't list all project templates.
+## Template options
 
-`-lang|--language {C#|F#|VB}`
+Each project template may have additional options available. The core templates have the following additional options:
 
-The language of the template to create. The language accepted varies by the template (see defaults in the [arguments](#arguments) section). Not valid for some templates.
+### console
 
-> [!NOTE]
-> Some shells interpret `#` as a special character. In those cases, you need to enclose the language parameter value, such as `dotnet new console -lang "F#"`.
+- **`-f|--framework <FRAMEWORK>`**
 
-`-n|--name <OUTPUT_NAME>`
+  Specifies the [framework](../../standard/frameworks.md) to target. Available since .NET Core 3.0 SDK.
 
-The name for the created output. If no name is specified, the name of the current directory is used.
+  The following table lists the default values according to the SDK version number you're using:
 
-`--nuget-source`
+  | SDK version | Default value   |
+  |-------------|-----------------|
+  | 3.1         | `netcoreapp3.1` |
+  | 3.0         | `netcoreapp3.0` |
 
-Specifies a NuGet source to use during install.
+- **`--langVersion <VERSION_NUMBER>`**
 
-`-o|--output <OUTPUT_DIRECTORY>`
+  Sets the `LangVersion` property in the created project file. For example, use `--langVersion 7.3` to use C# 7.3. Not supported for F#. Available since .NET Core 2.2 SDK.
```

Comment:
What's the default language version?

Reply:
Good question. @KathleenDollard? @BillWagner? do you know the mapping?

Reply:
I found the answer. I'll just link to that.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults